packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -111,10 +112,12 @@ export const start = async (zcf, privateArgs, baggage) => {
     },
   );
 
+  const withdrawFromSeat = vowTools.retriable('withdrawFromSeat', wFS);
```

Comment:
As you mentioned to me, `withdrawFromSeat` is very non-idempotent, so it wouldn't be a candidate retriable anyway. Am I missing something?